localstack/utils/container_networking.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+from functools import lru_cache
+from typing import Optional
+
+from localstack import config
+from localstack.utils.container_utils.container_client import ContainerException
+from localstack.utils.docker_utils import DOCKER_CLIENT
+
+LOG = logging.getLogger(__name__)
+
+
+@lru_cache
+def get_main_container_network() -> str:
```

Comment:
Technically this returns Optional[str]. If we fall into the except block main_container_network (and thus the returned value) will be None

Reply:
fixed!